_sass/bootstrap/_card.scss

```diff
@@ -159,6 +159,9 @@
       margin-right: $card-deck-margin;
       margin-bottom: 0; // Override the default
       margin-left: $card-deck-margin;
+      width: auto;
+      height: auto;
+      max-width: 300px;
     }
```

Comment:
Never change the source file of a framework or library. This was incredibly hard to chase down.